clang-tools-extra/clang-doc/HTMLGenerator.cpp

```diff
@@ -418,9 +419,13 @@ genRecordMembersBlock(const llvm::SmallVector<MemberTypeInfo, 4> &Members,
     if (Access != "")
       Access = Access + " ";
     auto LIBody = std::make_unique<TagNode>(HTMLTag::TAG_LI);
-    LIBody->Children.emplace_back(std::make_unique<TextNode>(Access));
-    LIBody->Children.emplace_back(genReference(M.Type, ParentInfoDir));
-    LIBody->Children.emplace_back(std::make_unique<TextNode>(" " + M.Name));
+    auto MemberDecl = std::make_unique<TagNode>(HTMLTag::TAG_DIV);
+    MemberDecl->Children.emplace_back(std::make_unique<TextNode>(Access));
+    MemberDecl->Children.emplace_back(genReference(M.Type, ParentInfoDir));
+    MemberDecl->Children.emplace_back(std::make_unique<TextNode>(" " + M.Name));
+    if (!M.Description.empty())
+      LIBody->Children.emplace_back(genHTML(M.Description));
+    LIBody->Children.emplace_back(std::move(MemberDecl));
```

Comment:
Is this required for Block commands? This seems like a separate change...

Reply:
I agree that this seems like separate change that should be its own PR.

Reply:
this originally meant to add support for block commands in member variables, I be happy to refactor these to its own pr though

Reply:
Yeah, lets split this up a bit. I'll leave the ordering up to you, but the other feature seems to be roughly independent.

Reply:
I've split the pr to https://github.com/llvm/llvm-project/pull/101255